requirements.txt

```diff
@@ -57,3 +57,23 @@ schedule==1.1.0
 #WS Messages
 websockets==10.4
 janus==1.0.0
+
+pytest~=7.2.0
+freqtrade~=2022.12.dev0
+filelock~=3.8.2
+plotly~=5.11.0
+setuptools~=65.6.3
+starlette~=0.22.0
+gym~=0.21.0
+torch~=1.13.1
+scikit-learn~=1.1.3
+scipy~=1.9.3
+xgboost~=1.7.2
+catboost~=1.1.1
+lightgbm~=3.3.3
+astor~=0.8.1
+ta~=0.10.2
+finta~=1.3
+tapy~=1.9.1
+matplotlib~=3.6.2
+PyYAML~=6.0
```

Comment:
this ... doesn't seem to be an intentional change.

It'll duplicate several dependencies we have in other requirements files (for example the plot and freqAI dependencies).

i think the only valid new dependency is `astor`. the others are either in other requirements files  (freqai, pytest, ...) - or freqtrade doesn't depend on them (finta).



We'll not install all possible ta libraries - nor will we force every dependency on every user (e.g.: torch is >500MB - and only needed if you use freqAI).